src/workerd/jsg/jsg.c++

```diff
@@ -272,6 +274,16 @@ void Lock::terminateExecution() {
   v8Isolate->TerminateExecution();
 }
 
+bool Lock::pumpMsgLoop() {
+  bool tasksRun = false;
+  // Ensure that this runs under limit enforcer scope
+  // We currently disable Minor GC, so this while loop will only ever run at max one iteration
+  while (IsolateBase::from(v8Isolate).pumpMsgLoop()) {
```

Comment:
I guess this loop is kind of redundant now, as the immediate caller also runs their own loop.